### DISCUSSION TITLE
Welcome to Full-Stack-Web-Development Discussions!

### DISCUSSION THREAD

**Discussion by ChandraPrakash-Bathula** [ChandraPrakash-Bathula/Full-Stack-Web-Development#5] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by OloyedeSOMA** [comment]
Hello!, how are you doing?
why did you close the community?

**Reply by ChandraPrakash-Bathula** [reply]
Hi @OloyedeSOMA thanks for the response on the discussion. I see no response and thought I could open in some day soon. Now, I'm going to re-open this.

**Reply by OloyedeSOMA** [reply]
thats great!

**Reply by OloyedeSOMA** [reply]
can i introduce myself properly now?

**Reply by ChandraPrakash-Bathula** [reply]
Yes, Go ahead @OloyedeSOMA

**Comment by OloyedeSOMA** [comment]
sorry, I was busy
okay uhm am a junior front-end web developer [still in training]xD no job neither internships yet Lol I have knowledge of html, css, js and vue.js. 
@ChandraPrakash-Bathula you?
